easybuild/easyconfigs/m/MPICH/MPICH-4.3.0-GCC-14.2.0-CUDA-12.8.0.eb

```diff
@@ -19,10 +19,11 @@ dependencies = [
 ]
 
 configopts = 'FFLAGS="-w -fallow-argument-mismatch -O2" '
-configopts += '--with-device=ch4:ucx --with-ucx=$EBROOTUCX '
-# Also build (separate) MPI ABI
-configopts += '--enable-mpi-abi '
+configopts += '--with-device=ch4:ucx '
```

Comment:
Shouldn't this also be a part of the updated easyblock then? Are there cases where it is _not_ used alongside `--with-ucx=$EBROOTUCX`